examples/meta/src/neural_nets/feedforward_net_classification.sg

```diff
@@ -0,0 +1,45 @@
+CSVFile f_feats_train("../../data/fm_train_real.dat")
+CSVFile f_feats_test("../../data/fm_test_real.dat")
+CSVFile f_labels_train("../../data/label_train_twoclass.dat")
+
+#![create_features]
+RealFeatures features_train(f_feats_train)
+RealFeatures features_test(f_feats_test)
+BinaryLabels labels_train(f_labels_train)
+#![create_features]
+
+#![add_layers]
```

Comment:
I assume this works in Python?
In that case, it is the grammar, which doesn't allow subsequent method calls.
Check if it works if you do this in a sequence of steps:
```
DynamicObjectArray all_layers = layers.input(num_feats)
all_layers = all_layers.linear(50)
all_layers = all_layers.softmax(2)
all_layers = all_layers.done()
```
 (or similar)